src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -295,6 +298,10 @@ public async Task<string> CreatePullRequestAsync(string repoUri, PullRequest pul
             };
             Octokit.PullRequest createdPullRequest = await Client.PullRequest.Create(owner, repo, pr);
 
+            // add reviewers to the created PR
+            PullRequestReviewRequest reviewers = new PullRequestReviewRequest(pullRequest.Reviewers.ToList());
+            await Client.PullRequest.ReviewRequest.Create(owner, repo, createdPullRequest.Number, reviewers);
```

Comment:
What happens here if an invalid name is passed? We probably should not fail to create the PR in that case

Reply:
Well we did create the review on line 299, but the caller is going to, presumably, retry (internet and such) not sure that that will do.

---
In reply to: [254760371](https://github.com/dotnet/arcade-services/pull/88#discussion_r254760371) [](ancestors = 254760371)